### PR TITLE
Various fix in modules (FQCN, version_added field)

### PR DESCRIPTION
**Pull request by bmillemathias**
Update the examples to use the FQCN for modules

### CODE REVIEW

File: plugins/modules/gluster_heal_info.py [hunk 1]
```diff
@@ -17,11 +17,11 @@
 module: gluster_heal_info
 short_description: Gather information on self-heal or rebalance status
 author: "Devyani Kota (@devyanikota)"
-version_added: "2.8"
+version_added: "1.0"
```
Comment (felixfontein) [inline plugins/modules/gluster_heal_info.py:20]:
```suggestion
```

File: plugins/modules/gluster_peer.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
   - Create or diminish a GlusterFS trusted storage pool. A set of nodes can be
     added into an existing trusted storage pool or a new storage pool can be
     formed. Or, nodes can be removed from an existing trusted storage pool.
-version_added: "2.6"
+version_added: "1.0"
```
Comment (felixfontein) [inline plugins/modules/gluster_peer.py:25]:
```suggestion
```

File: plugins/modules/gluster_volume.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
 short_description: Manage GlusterFS volumes
 description:
   - Create, remove, start, stop and tune GlusterFS volumes
-version_added: '1.9'
+version_added: '1.0'
```
Comment (felixfontein) [inline plugins/modules/gluster_volume.py:19]:
```suggestion
```

File: plugins/modules/gluster_volume.py [hunk 4]
```diff
@@ -47,20 +47,20 @@
     description:
       - Arbiter count for volume.
     type: int
-    version_added: '2.3'
+    version_added: '1.0'
```
Comment (felixfontein) [inline plugins/modules/gluster_volume.py:50]:
```suggestion
```

File: plugins/modules/gluster_volume.py [hunk 5]
```diff
@@ -47,20 +47,20 @@
...
   stripes:
     description:
       - Stripe count for volume.
     type: int
   disperses:
     description:
       - Disperse count for volume.
-    version_added: '2.2'
+    version_added: '1.0'
```
Comment (felixfontein) [inline plugins/modules/gluster_volume.py:58]:
```suggestion
```

File: plugins/modules/gluster_volume.py [hunk 6]
```diff
@@ -47,20 +47,20 @@
...
     type: int
   redundancies:
     description:
       - Redundancy count for volume.
-    version_added: '2.2'
+    version_added: '1.0'
```
Comment (felixfontein) [inline plugins/modules/gluster_volume.py:63]:
```suggestion
```

**Comment by pkesavap** [comment]
@kesavanvt

**Comment by kesavanvt** [comment]
LGTM

**Pull request merged**